documentation/asciidoc/computers/getting-started/network-installation.adoc

```diff
@@ -1,65 +1,28 @@
-== Installing over the Network (BETA)
+== Installing over the Network
 
-WARNING: Network installation is in currently in PUBLIC BETA. Please do not rely on it in production.
+WARNING: Network installation is in currently only available in the LATEST/STABLE bootloader build.
 
 When you get a new Raspberry Pi, you also need an SD Card with an operating system installed on it. The easiest way to get the operating system onto the SD Card is to use the https://www.raspberrypi.com/software/[Raspberry Pi Imager] application running on another computer to xref:getting-started.adoc#installing-the-operating-system[to copy it to your SD Card].
 
 But how do you get the operating system onto your SD card if you don’t have another computer?
 
 If you do not have another computer to run Raspberry Pi Imager you can start the application directly on a Raspberry Pi 4 or Raspberry Pi 400 by connecting it to the Internet with an Ethernet cable. This will allow you to install the operating system onto a **blank** SD Card directly from the network without using another computer.
 
-IMPORTANT: Network installation is only supported on Raspberry Pi 4 and Raspberry Pi 400. Additionally, to avoid slowing down “normal boot,” the network installation feature is only enabled if there’s no bootable media present, and you have xref:raspberry-pi.adoc#usb-mass-storage-boot[USB mass storage boot] in your xref:raspberry-pi.adoc#BOOT_ORDER[boot order].
-
-=== Updating to the BETA bootloader
```

Comment:
I think we should leave something here to tell folks how to update to the latest stable bootloader build, if this isn't the way to do that anymore we should document — perhaps as a NOTE — how to do that. Most Raspberry Pi's out in the world won't have the correct bootloader so without this section folks could be confused as to how to get it working.

Reply:
http://downloads.raspberrypi.org/os_list_imagingutility_v3.json has "factory default" and "Beta" in the eeprom-image section, so I guess that means that `rpi-eeprom-updare` is the recommended way to update to the "latest" bootloader image?

Reply:
Thanks!

Reply:
I added a link to a new section on the bootloader page

Reply:
Hmmmm, is it worth removing the "Beta bootloader" option from RPi Imager again, now that network-boot is in "latest" ?